### ISSUE TITLE
Continue to train from pretrained NER model

### ISSUE THREAD

**Issue by muzaluisa** [jouniluoma/keras-bert-ner#5]
Hey,

Thank you for creating this code. I have been using your NER system to train from scratch, but wanted to know if further training from already trained NER model is supported? And if yes, what is the script I need to investigate? 

In ner.py you have: 

`pretrained_model, tokenizer = load_pretrained(args)`

but load_pretrained seems to load only bert basic model and not ner model:

```
def load_pretrained(options):
    model = load_trained_model_from_checkpoint(
        options.bert_config_file,
        options.init_checkpoint,
        training=False,
        trainable=True,
        seq_len=options.max_seq_length,
    )
    tokenizer = tokenization.FullTokenizer(
        vocab_file=options.vocab_file,
        do_lower_case=options.do_lower_case
    )
    return model, tokenizer

```

Kind regards,
Luiza

**Comment by muzaluisa** [comment]
I figured out how to do it myself, so you can close the issue :) 
I ner.py instead of creating ner model, I load existing one model and do some several other modifications to be sure I am aligned with existing model labels.
`ner_model, tokenizer, labels, config = load_ner_model(args.ner_pretrained_model_dir)`

**Comment by jouniluoma** [comment]
Good to hear that you got it working! I have not that often tried to train the model again, just used it for inference. Anyhow, that is the way to load the saved model.
